src/Core/Kernel.php

```diff
@@ -351,7 +351,7 @@ private function instantiatePlugins(array $plugins): void
             }
 
             /** @var Plugin $plugin */
-            $plugin = new $className((bool) $pluginData['active']);
+            $plugin = new $className((bool) $pluginData['active'], $this->getProjectDir() . $pluginData['path']);
```

Comment:
a slash is missing between the pathes, because of this commit: https://github.com/shopware/platform/commit/b36982eb3241066e9ff7f30956d86ea3d8cfd7ac

Reply:
Thanks for the heads-up!

Is there a way to test this part of the logic?